service/chartconfig/v1/helm/error.go

```diff
@@ -8,6 +8,7 @@ import (
 
 const (
 	releaseNotFoundErrorPrefix = "No such release:"
+	releaseNotFoundErrorSuffix = "not found"
```

Comment:
This is a pain because the error string for ReleaseHistory is different and harder to match for.

`rpc error: code = Unknown desc = release: "release-missing" not found`

Reply:
We should try to engage upstream and make people aware of this and inspire how we do error handling. Like we have a matcher for every fucking error and using open source libraries without that is so awful.